src/main/java/com/mongodb/jdbc/oidc/manualtests/TestOidcCallbackWithBadRefreshToken.java

```diff
@@ -16,26 +16,25 @@
 
 package com.mongodb.jdbc.oidc.manualtests;
 
-import com.mongodb.jdbc.oidc.OidcCallback;
-import com.mongodb.jdbc.oidc.OidcCallbackContext;
-import com.mongodb.jdbc.oidc.OidcCallbackResult;
-import javax.security.auth.RefreshFailedException;
+import com.mongodb.MongoCredential.OidcCallback;
+import com.mongodb.MongoCredential.OidcCallbackContext;
+import com.mongodb.MongoCredential.OidcCallbackResult;
+import com.mongodb.jdbc.oidc.JdbcOidcCallback;
+import com.mongodb.jdbc.oidc.JdbcOidcCallbackContext;
 
 public class TestOidcCallbackWithBadRefreshToken {
 
     public static void main(String[] args) {
-        OidcCallback oidcCallback = new OidcCallback();
+        OidcCallback oidcCallback = new JdbcOidcCallback();
 
         String badRefreshToken = "bad-refresh-token";
         OidcCallbackContext context =
-                new OidcCallbackContext(null, 1, badRefreshToken, TestOidcUtils.IDP_INFO);
+                new JdbcOidcCallbackContext(null, 1, badRefreshToken, TestOidcUtils.IDP_INFO);
 
         try {
             OidcCallbackResult result = oidcCallback.onRequest(context);
             System.out.println("This should not print, bad refresh token expected to fail.");
             System.out.println(result);
-        } catch (RefreshFailedException e) {
```

Comment:
I think that you should keep this check. You'll need to catch Runtime exceptions and check the cause. It should be an instance of RefreshFailedException or it means that the exception thrown is not what we expect, and therefore the test should "fail".

Reply:
Sure thing, I was being lazy when I updated the `JdbcOidcCallback` class to remove the `throws` and then removed the corresponding catch here.  I added the check for `instance of RefreshFailedException`